Api-Versioning/src/BookStore.Application/Books/BookAppService.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Threading.Tasks;
+using Volo.Abp.ApiVersioning;
+using Volo.Abp.Domain.Entities;
+
+namespace BookStore.Books;
+
+public class BookAppService : BookStoreAppService, IBookAppService
+{
+    public Task<BookDto> GetAsync()
+    {
+        return Task.FromResult(
+            new BookDto
+            {
+                Title = "Mastering ABP Framework",
+                ISBN = "978-1-80107-924-2"
+            }
+        );
+    }
+}
+
+public class BookV2AppService : BookStoreAppService, IBookV2AppService
+{
+    public Task<BookDto> GetAsync()
+    {
+        return Task.FromResult(
+            new BookDto
+            {
+                Title = "Mastering ABP Framework V2",
+                ISBN = "978-1-80107-924-2"
+            }
+        );
+    }
+
+    public Task<BookDto> GetAsync(string isbn)
+    {
+        if (isbn == "978-1-80107-924-2")
+        {
+            return Task.FromResult(
+                new BookDto
+                {
+                    Title = "Mastering ABP Framework V2",
+                    ISBN = "978-1-80107-924-2"
+                }
+            );
+        }
+
+        throw new EntityNotFoundException();
+    }
+}
+
+public class BookV3AppService : BookStoreAppService, IBookV3AppService
+{
+    public Task<BookDto> GetAsync()
+    {
+        return Task.FromResult(
+            new BookDto
+            {
+                Title = "Mastering ABP Framework V2",
```

Comment:
change `V2` to `V3`?
also line 71

Reply:
**This pull request is still a work in progress!**